I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 1993, in script
    show sa home_annoyed at tope
  File "game/script.rpy", line 1993, in <module>
    show sa home_annoyed at tope
NameError: name 'tope' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 1993, in script
    show sa home_annoyed at tope
  File "C:\Users\Usuario\Documents\Renpy\renpy\ast.py", line 1408, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "C:\Users\Usuario\Documents\Renpy\renpy\ast.py", line 1367, in show_imspec
    at_list = [ renpy.python.py_eval(i) for i in at_list ]
  File "C:\Users\Usuario\Documents\Renpy\renpy\ast.py", line 1367, in <listcomp>
    at_list = [ renpy.python.py_eval(i) for i in at_list ]
  File "C:\Users\Usuario\Documents\Renpy\renpy\python.py", line 1092, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\Usuario\Documents\Renpy\renpy\python.py", line 1085, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 1993, in <module>
    show sa home_annoyed at tope
NameError: name 'tope' is not defined

Windows-10-10.0.19045 AMD64
Ren'Py 8.0.3.22090809
The Working Woman's Guide to Burning Bridges 1.0
Fri Jun 30 09:05:38 2023
